docs/essentials/platform-feature-support.md

```diff
@@ -21,6 +21,7 @@ Xamarin.Essentials supports the following platforms and operating systems:
 | UWP | 10.0.16299.0 or higher |
 
 > [!NOTE]
+>
```

Comment:
I'm not sure it's required here? or if it is, the markdown linter is not dealing well with our markdown extension. Going to test on an internal page that I can preview, to ensure rendering is unaffected.

Reply:
Sounds good. The dotnet/docs verified the "checklist" blockquotes before, but good to verify to see if there is any differences in the build setup

Reply:
Confirmed, looks identical
![Screen Shot 2019-09-16 at 1 36 49 PM](https://user-images.githubusercontent.com/199373/64991670-21c11f80-d887-11e9-9201-013e66cd63b8.png)